draft-ietf-ppm-dap.md

```diff
@@ -3785,12 +3789,12 @@ unauthenticated parties. Thus there are some threats that DAP does not defend
 against and which are considered outside of its threat model. These and others
 are enumerated below, along with potential mitigations.
 
-Attacks on robustness:
+Attacks on verifiability:
 
-1. Aggregators can defeat robustness by emitting incorrect aggregate shares, by
-   omitting reports from the aggregation process, or by manipulating the VDAF
+1. Aggregators can defeat verifiability by emitting incorrect aggregate shares,
+   by omitting reports from the aggregation process, or by manipulating the VDAF
    preparation process for a single report. DAP follows VDAF in providing
-   robustness only if both Aggregators honestly follow the protocol.
+   verifiability only if both Aggregators honestly follow the protocol.
```

Comment:
Suggested rewording: "Like the underlying VDAF, DAP only ensures correct computation of the aggregate result if both Aggregators correctly execute the protocol."